### PR TITLE
Add a note about custom logs vs devtools logs

### PR DESCRIPTION
**Pull request by KevinU3**
Added more notes to inform developers that uses an online IDE about the location of the timing logs.

### Description

Some online IDE does not log the timing methods on their own console. Suggested opening DevTools.

### Motivation

Inform developers who uses an online IDE on where to look for the timing logs.

### CODE REVIEW

**Comment by Elchi3** [comment]
@KevinU3 Thanks for your PR! 

"the timing methods do not log to the console of some online IDE."

I think "some online IDEs" can always decide how they implement the console standard. I'm not sure how useful the note is. And while this is true for the IDE you have in mind and for the timing methods there, I guess it could be generalized to say that certain websites/online IDEs re-implement `console` objects and that they don't necessarily come with all of the features that are in the browser's DevTools implementation of `console`. Does that make sense to you?

**Comment by KevinU3** [comment]
> @Elchi3 Thanks for getting back to me!
> 
> "don't necessarily come with all of the features that are in the browser's DevTools implementation of `console`"

While I agree with this, I think it is still best to be explicit about stating the reason for the logs not displaying (this could be the reason you stated) and how to resolve it (find the logs). This is especially important in the "guide" section of the console API doc. Don't you agree?

**Comment by Elchi3** [comment]
Yes, I think it's a good idea to note this! Should we then make the note box a bit more generic and place it at the top rather than in a section specific to timers? I would expect a wording like this to explain the situation (feel free to improve):

> Note: Certain online IDEs and editors re-implement the `console` API and behave similar to the browser's `console` object. However, not all functionality might be provided there (for example, you might not get correct timing logs). Open the browser's DevTools to access the browser's `console` API as documented here.

File: files/en-us/web/api/console/index.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ gives a few [Usage](#usage) examples.
 
 {{AvailableInWorkers}}
 
+> **Note:** Certain online IDEs and editors may implement the console API differently than the browsers. As a result, certian functionality of the console API such as the timer methods, may not be outputted in the console of some online IDE or editors. Open your browser's DevTools console to see the logs as shown in this documentation.
```
Comment (Elchi3) [inline files/en-us/web/api/console/index.md:29]:
```suggestion
> **Note:** Certain online IDEs and editors may implement the console API differently than the browsers. As a result, certain functionality of the console API, such as the timer methods, may not be outputted in the console of online IDEs or editors. Always open your browser's DevTools console to see the logs as shown in this documentation.
```

File: files/en-us/web/api/console/index.md [hunk 2]
```diff
@@ -242,6 +244,7 @@ Notice that the timer's name is displayed both when the timer is started and whe
 > **Note:** It's important to note that if you're using this to log the timing for network traffic, the timer will report the total time for the transaction, while the time listed in the network panel is just the amount of time required for the header.
 > If you have response body logging enabled, the time listed for the response header and body combined should match what you see in the console output.
 
+
```
Comment (Elchi3) [inline files/en-us/web/api/console/index.md:247]:
```suggestion
```

**Review by Elchi3 — APPROVED** [review]
Thank you! 👍

**Pull request merged**